bot/code_review_bot/revisions.py

```diff
@@ -188,6 +188,52 @@ def from_try_task(try_task: dict, phabricator: PhabricatorAPI):
         # Load target patch from Phabricator for Try mode
         patch = phabricator.load_raw_diff(diff_id)
 
+        # The parent decision task should exist
+        assert decision_task is not None, "Missing parent decision task"
+        logger.info("Found decision task", name=decision_task["metadata"]["name"])
+
+        # Match the decision task environment to get the mercurial information
+        decision_env = decision_task["payload"]["env"]
+        head_repository = (
+            base_repository
+        ) = head_changeset = base_changeset = repository_try_name = None
+        for prefix in settings.decision_env_prefixes:
+            head_repository_key = f"{prefix}_HEAD_REPOSITORY"
+            base_repository_key = f"{prefix}_BASE_REPOSITORY"
+            head_changeset_key = f"{prefix}_HEAD_REV"
+            base_changeset_key = f"{prefix}_BASE_REV"
+            if (
+                head_repository_key not in decision_env
+                or base_repository_key not in decision_env
+                or head_changeset_key not in decision_env
+                or base_changeset_key not in decision_env
+            ):
+                continue
+
+            head_repository = decision_env[head_repository_key]
+            base_repository = decision_env[base_repository_key]
+            head_changeset = decision_env[head_changeset_key]
+            base_changeset = decision_env[base_changeset_key]
+            repository_try_name = head_repository.rstrip("/").rsplit("/", 1)[-1]
```

Comment:
We do not want to use the maxsplit=1 here (in the case of `hg.mozilla.org/ci/ci-configuration-try` we want `ci-configuration-try` not `ci/ci-configuration-try`). 

A better approach would be to use urrlib.parse.urlparse, then get the path attribute, then split on `/`, remove the empty strings, and keep the last one.

Code golf:

```
list(filter(None, urllib.parse.urlparse(head_repository).path.split('/')))[-1]
```